src/pages/Opening/Opening.jsx

```diff
@@ -0,0 +1,246 @@
+import { gql } from 'apollo-boost';
+import React, { Component, Fragment } from 'react';
+import { Mutation, Query } from 'react-apollo';
+import { Button, DataTable, TableHeader, TableBody, TableRow, TableColumn, Snackbar, MenuButtonColumn, FontIcon } from 'react-md';
+import { withRouter } from 'react-router-dom';
+import OpeningDialog from './Dialog';
+import Loading from '../../components/Loading';
+import NumberFormat from 'react-number-format';
+import ConfirmDialog from '../../components/ConfirmDialog';
+
+class Opening extends Component {
```

Comment:
let's use `PropTypes` to specify the inputs for each component

Reply:
PropTypes Added